HLTrigger/Configuration/python/Tools/confdb.py

```diff
@@ -193,6 +193,13 @@ def specificCustomize(self):
 %%(process)s = customizeHLTforCMSSW(%%(process)s,"%s")
 """ % (self.config.type)
 
+    # Eras-based customisations
+    self.data += """
+# Eras-based customisations
+from HLTrigger.Configuration.Eras import modifyHLTforEras
+modifyHLTforEras(%%(process)s)
```

Comment:
Please remove one of the `%` in the above line - I had to do so for 90X in #17096